components/brave_wallet/browser/eth_tx_controller.cc

```diff
@@ -383,38 +383,49 @@ void EthTxController::ApproveHardwareTransaction(
   }
 }
 
+void EthTxController::GetTransactionMessageToSign(
+    const std::string& tx_meta_id,
+    GetTransactionMessageToSignCallback callback) {
+  std::unique_ptr<EthTxStateManager::TxMeta> meta =
+      tx_state_manager_->GetTx(tx_meta_id);
+  if (!meta) {
+    LOG(ERROR) << "No transaction found";
```

Comment:
this should be a VLOG and needs more context in the message. What transaction?

Reply:
done